documentation/API_documentation/simple-edge-discovery_API_Readiness_Checklist.md

```diff
@@ -1,19 +1,19 @@
 # API Readiness Checklist
 
-Checklist for simple-edge-discovery v0.11.0 in r0.11.0-rc.1
+Checklist for simple-edge-discovery v0.11.0-rc.1 in r1.1
 
 | Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
 |----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
-|  1 | API definition                               |   M   |         M         |    M    |    M   |  Y   | [link](https://github.com/camaraproject/SimpleEdgeDiscovery/blob/main/code/API_definitions/simple-edge-discovery.yaml) |
+|  1 | API definition                               |   M   |         M         |    M    |    M   |  Y   | [link](/code/API_definitions/simple-edge-discovery.yaml) |
 |  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |  Y   |      |
 |  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   |      |
 |  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |  Y   |      |
-|  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | [link](https://github.com/camaraproject/SimpleEdgeDiscovery/blob/main/code/API_definitions/simple-edge-discovery.yaml) |
-|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](https://github.com/camaraproject/SimpleEdgeDiscovery/tree/main/documentation/API_documentation/SED_User_Story.md) |
-|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [link](https://github.com/camaraproject/SimpleEdgeDiscovery/tree/main/code/Test_definitions) |
-|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |  N   |  |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | [link](/code/API_definitions/simple-edge-discovery.yaml) |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](/documentation/API_documentation/SED_User_Story.md) |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [link](/code/Test_definitions) |
```

Comment:
The feature still refers to version 0.10.0. 
Question: is it necessary to reference the API version ? as it anyway is in the same release package ?

Reply:
Good point, I've removed it.